platform/viewer/src/index.js

```diff
@@ -43,6 +44,7 @@ const appProps = {
     OHIFDicomHtmlExtension,
     OHIFDicomMicroscopyExtension,
     OHIFDicomPDFExtension,
+    OHIFLesionTrackerExtension,
```

Comment:
If you've pulled in latest, only have this in the `index-lesion-tracker.js` file, and you can test using `yarn run dev:lt`

Reply:
Great, updated, thanks.